docs/Installation.md

```diff
@@ -65,6 +65,10 @@ sudo cpan -i Zonemaster::Backend
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
+Add Zonemaster user:
+sudo useradd zonemaster

```

Comment:
Replace with the following and move before line 63.

*

Add Zonemaster user that will run Zonemaster Backend:

```
sudo useradd zonemaster
```